packages/markdown-ooxml/src/OoxmlTransformer.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const xmljs = require('xml-js');
+const typeOf = require('type-of');
+const { NS_PREFIX_CommonMarkModel } = require('@accordproject/markdown-common').CommonMarkModel;
+
+/**
+ * Transforms OOXML to CiceroMark
+ */
+class OoxmlTransformer {
+    /**
+     * Gets the id of the variable
+     * 
+     * @param {Array} elements the variable elements
+     * @returns {string} the name/id of the variable
+     */
+    getId(elements) {
+        const variableProperties = elements[0]
+        for (const property of variableProperties.elements) {
+            if (property.name === 'w:tag') {
+                return property.attributes['w:val'];
+            }
+        }
+    }
```

Comment:
Is there a case where `getId()` will not return anything, and you'd want to have a base case after the for loop?

Reply:
@irmerk I think every variable will have a corresponding ID so it ought to return something. And I am only calling this function when I parse `w:sdt` which are node types for content controls.